### DISCUSSION TITLE
Why did you decide to put "wc:"

### DISCUSSION THREAD

**Discussion by pikaboo** [WalletConnect/walletconnect-monorepo#756] [Developer Support]
QRs generated with this type of code `wc:8372fd76-b531-4a41-a801-a97732c5c7cc@1?bridge=https%3A%2F%2Fbridge.walletconnect.org&key=8be7078bd985bca318eef6e3069845437335531461abc0db60ed47d089349435` behind them are not readable to the built in camera app, only for custom apps, so the app cannot be redirected using the default scenario of URL deep linking

According to android documentation, the scheme must be anything:// 

so my scheme in this case would be "anything" but since there is "://" it would work.
And the "wc:..." link does not work because it is missing "//" in it

**Comment by benma** [comment]
I don't think `://` is required for registering the URI. 

Example:

https://github.com/digitalbitbox/bitbox-wallet-app/blob/b0bf3924546e4bb4a58fabbf6676cf630e0091a0/frontends/android/BitBoxApp/app/src/main/AndroidManifest.xml#L45

This registers `aopp:?...` URIs tha also don't have `://` and the app is launched correctly when scanning such a code (e.g. from https://demo.aopp.group/).

**Comment by pikaboo** [comment]
It does not read the URI correctly. It does not read the URI **at all**. I was trying to use the built in camera app to read the URI. but it seems the camera app is not even trying to read it.

**Comment by justindg** [comment]
Was there ever a solution to this? I'm also trying to launch my app via the default/native Android camera app, but it only reads the `wc:` link as plain text
